### PR TITLE
fixed issue with attempting to load data from an empty directory

### PR DESCRIPTION
**Pull request by joshuaherrera**
Resolves issue #95 by checking that the latest directory contains at least 1 file.

### CODE REVIEW

File: core/record.go [hunk 1]
```diff
@@ -170,7 +170,10 @@ func (t *Timetrace) LoadLatestRecord() (*Record, error) {
 		return nil, nil
 	}
 
-	dir := latestDirs[len(latestDirs)-1]
+	dir, err := t.latestNonEmptyDir(latestDirs)
+	if err != nil {
+		return nil, err
+	}
```
Comment (dominikbraun) [inline core/record.go:176]: I suggest returning `ErrTrackingNotStarted` to indicate that the user has no start tracking first.

```suggestion
	if err != nil {
		if errors.Is(err, ErrAllDirectoriesEmpty) {
			return nil, ErrTrackingNotStarted
		}
		return nil, err
	}
```

**Review by dominikbraun — CHANGES_REQUESTED** [review]
Nice solution! Maybe we'll extend the `Filesystem` interface later and add a function like `LatestRecordDirPath` so we can move this filesystem-related logic into the `fs` package.

Otherwise, I just have a small suggestion 😄

**Comment by joshuaherrera** [comment]
That change looks good, I just committed our suggestion.

> Maybe we'll extend the `Filesystem` interface later and add a function like `LatestRecordDirPath` so we can move this filesystem-related logic into the `fs` package.

Yes, I wasn't sure where the best place to put these helper functions would be, so I put them in the timetrace.go file. I like this idea better though.

**Review by dominikbraun — APPROVED** [review]
LGTM 👍

**Comment by dominikbraun** [comment]
This fix is contained in [release 0.7.1](https://github.com/dominikbraun/timetrace/releases/tag/v0.7.1).

**Pull request merged**